### ISSUE TITLE
var-file sent to terraform is missing default config-location

### ISSUE THREAD

**Issue by mblakele** [nadnerb/terraform_exec#12]
When using the default config location, `terraform_exec` doesn't seem to pass the right var-file to `terraform`: it's `/staging.tfvars` instead of `config/staging.tfvars`.

```
$ terraform_exec --version
Terraform Exec version 0.0.5
$ terraform_exec refresh staging
[staging]

Execute Terraform command
Command:     refresh
Environment: staging

Using default config location: /Users/mblakele/Source/terraform-projects/fubar/config
Using terraform config: /Users/mblakele/Source/terraform-projects/fubar/config/staging.tfvars

AWS credentials
s3 bucket:  baz-ops
s3 key:     terraform/state/staging/fubar
aws region: us-west-2

Syncing project state with S3
Downloading project state: baz-ops/terraform/state/staging/fubar/tfstate/staging/terraform.tfstate to: ./tfstate/staging/terraform.tfstate

Downloaded successfully from S3

---------------------------------------------
terraform refresh -var-file /staging.tfvars -state=./tfstate/staging/terraform.tfstate -var environment=staging -var config_location=
---------------------------------------------

invalid value "/staging.tfvars" for flag -var-file: Error reading /staging.tfvars: open /staging.tfvars: no such file or directory
Usage: terraform refresh [options] [dir]
```

Workaround: `terraform_exec refresh staging --config-location=config`

**Comment by nadnerb** [comment]
This looks like a regression, thanks for pointing it out, I will aim to have a fix asap. Unfortunately I am not using terraform at the moment but definitely will aim to fix this.

**Comment by nadnerb** [comment]
This should be fixed in 0.0.7, let me know if you have any further issues.
